src/components/ProvideDetails.jsx

```diff
@@ -125,7 +125,8 @@ const provideDetails = props => {
 			response => {
 
 				try {
-					const address = response.data.address.Match_addr;
+					let address = _.split(response.data.address.Match_addr, ',', 3);
```

Comment:
Instead of using underscore, we can use javascript's built in string [function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split)

it would look something like

```js
response.data.address.Match_addr.split(',');
```

this avoids specifying the limit, which we don't want, because theoretically this function could be used for any array of strings.